numpy/core/tests/test_multiarray.py

```diff
@@ -9989,3 +9995,35 @@ def test_sort_uint():
 
 def test_private_get_ndarray_c_version():
     assert isinstance(_get_ndarray_c_version(), int)
+
+
+@pytest.mark.parametrize("N", np.arange(1, 512))
+@pytest.mark.parametrize("dtype", ['e', 'f', 'd'])
+def test_argsort_float(N, dtype):
+    np.random.seed(42)
```

Comment:
For preference, create a `RandomState` object and call its methods rather than use the `np.random.sample`, etc. functions. We also prefer non-joke seeds.

```suggestion
    rnd = np.random.RandomState(1720092927)
```

Reply:
Sure.